utils/featureFlagExist.tsx

```diff
@@ -0,0 +1,35 @@
+import { SourceFile } from "ts-morph";
+import { getDefaultExportedObject } from "./tsMorph/getDefaultExportedObject";
+import { toObjectValue } from "./tsMorph/toObjectValue";
+
+type FeatureFlagExistResult = {
+    exist: boolean;
+    hasError: boolean;
+};
+
+/*
+ * Check if feature flag exist in the featureFlag object
+ */
+export const featureFlagExist = (source: SourceFile, name: string): FeatureFlagExistResult => {
+    const projectObjectLiteral = getDefaultExportedObject(source);
+
+    if (!projectObjectLiteral) {
+        return {
+            exist: false,
+            hasError: true
+        };
+    }
+
+    const projectObject = toObjectValue(projectObjectLiteral);
+    if (projectObject?.featureFlags?.[name]) {
```

Comment:
Previous comment applies here as well.

Reply:
BTW if `projectObject` exists, I guess there's no need for `?.featureFlags`. `.featureFlags` should also be good.

Reply:
I have added this check for any case. This is the file from `5.36.1 ` version of webiny. I don't have any feature flags here.

![image](https://github.com/webiny/webiny-upgrades/assets/82515066/71a620db-6be1-4d17-8495-98b5ce87a15e)

This file can be changed by the user as well, everything can happen. I will remove the first check, but, I will leave the second question mark.
Let me know if I am wrong with the `webiny.project.ts` file maybe I am missing something.